src/modules/75-cf/strings/strings.en.yaml

```
@@ -1966,7 +1934,7 @@ onboarding:
       on a emulator
       ```shell
       cd examples/getting_started
-      export FF_API_KEY=<your api key>
+      export FF_API_KEY={{apiKey}}
 
       flutter pub get
       flutter run --dart-define=FF_API_KEY=$FF_API_KEY -d <device id>
```

I'd imagine this needs to drop the `--dart-define` part (please consult a flutter expert though)
```suggestion
      flutter run -d <device id>
```

---

I think you are correct so I have removed, I have asked in our help engineering channel and if any issues arise I will address